zerver/views/documentation.py

```diff
@@ -146,13 +146,27 @@ def add_integrations_context(context: Dict[str, Any]) -> None:
     context['integrations_dict'] = alphabetical_sorted_integration
     context['integrations_count_display'] = integrations_count_display
 
+def add_integrations_open_graph_context(context: Dict[str, Any], request: HttpRequest) -> None:
+    path_name = request.path.rstrip('/').split('/')[-1]
+    if path_name in INTEGRATIONS:
+        integration = INTEGRATIONS[path_name]
+        context['OPEN_GRAPH_TITLE'] = 'Zulip Integration - %s' % (integration.display_name,)
+        context['OPEN_GRAPH_DESCRIPTION'] = (
+            'Instructions to configure a Zulip integration for %s' % (integration.display_name,))
```

Comment:
Maybe "the Zulip integration for"?

Reply:
How about the following?

title: Connect %s to Zulip
description: Zulip comes with over a hundred native integrations out of the box, and integrates with Zapier, IFTTT, and Hubot to provide hundreds more. Connect the apps you use everyday to Zulip.

Reply:
Looks reasonable to me. Updating the PR.